<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../图标-网上下载/ico/Documents Folder.ico</normaloff>../../../../../图标-网上下载/ico/Documents Folder.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menu_W">
    <property name="title">
     <string>窗口(W)</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionCloseAll"/>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助(H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_W"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionUndo"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="actionTile"/>
   <addaction name="actionCascade"/>
   <addaction name="actionCloseAll"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/新建.png</normaloff>../../../../../图标-网上下载/图标2-alibba/新建.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/打开.png</normaloff>../../../../../图标-网上下载/图标2-alibba/打开.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/保存.png</normaloff>../../../../../图标-网上下载/图标2-alibba/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存文件(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/另存为.png</normaloff>../../../../../图标-网上下载/图标2-alibba/另存为.png</iconset>
   </property>
   <property name="text">
    <string>另存为(A)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/撤销.png</normaloff>../../../../../图标-网上下载/图标2-alibba/撤销.png</iconset>
   </property>
   <property name="text">
    <string>撤销(U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/恢复.png</normaloff>../../../../../图标-网上下载/图标2-alibba/恢复.png</iconset>
   </property>
   <property name="text">
    <string>恢复(R)</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切(T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴(P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭(O)</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/关闭.png</normaloff>../../../../../图标-网上下载/图标2-alibba/关闭.png</iconset>
   </property>
   <property name="text">
    <string>关闭所有(A)</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/平铺.png</normaloff>../../../../../图标-网上下载/图标2-alibba/平铺.png</iconset>
   </property>
   <property name="text">
    <string>平铺显示(T)</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/层叠.png</normaloff>../../../../../图标-网上下载/图标2-alibba/层叠.png</iconset>
   </property>
   <property name="text">
    <string>层叠显示(C)</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/下张.png</normaloff>../../../../../图标-网上下载/图标2-alibba/下张.png</iconset>
   </property>
   <property name="text">
    <string>下一个(X)</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/上张.png</normaloff>../../../../../图标-网上下载/图标2-alibba/上张.png</iconset>
   </property>
   <property name="text">
    <string>前一个(V)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../图标-网上下载/图标2-alibba/问题.png</normaloff>../../../../../图标-网上下载/图标2-alibba/问题.png</iconset>
   </property>
   <property name="text">
    <string>关于(A)</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>关于Qt(Q)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
